provisioning/roles/mangaki_sync/tasks/main.yml

```diff
@@ -1,4 +1,16 @@
 ---
+- name: Ensure static files directory to be present
+  file:
+    owner: '{{ mangaki_sync_user }}'
+    group: '{{ mangaki_sync_group }}'
+    mode: 'u=rwx,g=rx,o=rx'
+    path: '{{ mangaki_source_static_root }}'
+    state: 'directory'
+  become: true
+  run_once: true
```

Comment:
We don't want `run_once` as this would limit the creation of the directory to a single machine in a distributed setup. This is idempotent, so let's just do it always, and it won't do anything, most of the time.